modules/seqexec/server/src/main/scala/seqexec/server/tcs/TcsEpics.scala

```diff
@@ -786,14 +810,19 @@ final class TcsEpicsImpl[F[_]: Async](epicsService: CaService, tops: Map[String,
   // for the in-position to change to true and stay true for stabilizationTime. It will wait up to `timeout`
   // seconds for that to happen.
   override def waitInPosition(stabilizationTime: Duration, timeout: FiniteDuration)(implicit T: Timer[F]): F[Unit] =
-    T.sleep(FiniteDuration(tcsSettleTime.toMillis, TimeUnit.MILLISECONDS)) *>
-      Sync[F].delay(filteredInPositionAttr.restart(stabilizationTime))
-        .flatMap(waitForValueF(_, "TRUE", timeout,"TCS inposition flag"))
+    T.sleep(FiniteDuration(tcsSettleTime.toMillis, TimeUnit.MILLISECONDS)) *> (
```

Comment:
I guess this is the core fix

Reply:
Yes it is.

Reply:
Yes and no. It is an improvement that would have fixed the problem, but the only case when this was called with a zero stabilization time was when there was no offset applied, and that case does not apply anymore.